03/app.js

```diff
@@ -42,17 +42,27 @@ books.getAuthor = function(isbn) {
 }
 
 books.getTitle = function(isbn, lang) {
-
+    const title = this[isbn]['title'];
+    for (const key in title) {
+        // console.log(key, title[key])
+        if (lang === key) {
+            return title[key]
+        }
+    }
 }
 
 books.getTranlator = function(isbn, lang) {
-
+    const translator = this[isbn]['translator'];
```

Comment:
tutaj jest błąd bardziej `this[isbn][lang];`, ale napierw powinieneś sprawdzić czy `this[isbn]` istnieje - możesz zrobić to przy pomocy `typeof this[isbn] !== 'undefined'`